sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md

```diff
@@ -14,6 +10,7 @@
 #### Dependency Updates
 
 - Upgraded from `org.apache.httpcomponents:httpclient:4.5.14` to `org.apache.httpcomponents.client5:httpclient5:5.4.3`.
```

Comment:
Instead of upgraded, we should mention that we replaced it with a different package as they are not the same artifact.